recipes/arb-bio/6.1.rev16223/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "ARB" %}
+{% set version = "6.1.rev16223" %}
```

Comment:
Can you double check that this version is correct in the conda way and that the next version will be called 6.2 or 7.0?

```
>>> LooseVersion("6.1.rev16223") < LooseVersion("6.2")
True
>>> LooseVersion("6.1.rev16223") < LooseVersion("6.1")
False

```